lms/djangoapps/instructor_analytics/tests/test_basic.py

```diff
@@ -26,7 +26,8 @@
     sale_record_features, sale_order_record_features, enrolled_students_features,
     course_registration_features, coupon_codes_features, list_may_enroll,
     AVAILABLE_FEATURES, STUDENT_FEATURES, PROFILE_FEATURES,
-    get_proctored_exam_results)
+    get_proctored_exam_results,
```

Comment:
Whenever we have to add new/forked `import` statements, it makes our lives easier to add them on a separate line.

```python
from instructor_analytics.basic import student_response_rows
```

An alternate solution would be to place the new import on its one line within the parens of the existing import, but _not_ place it at the end, removing the need for adding a comma after `get_proctored_exam_results`.

This both reduces the size of the diff, as well as lowering its complexity which means it's easier to review and less likely to cause conflicts. Changesets with _both_ additions and deletions are more troublesome to maintain than a simple addition.

Specifically, if edX ever adds another import after `get_proctored_exam_results`, we'll have a conflict. If it's on it's own line (and not added to the very end of the existing statement), it will always merge cleanly.

I already have a WIP branch to update many similar, pre-existing changes; I'd much prefer if we stop the bleeding and not introduce any new occurrences that I'll have to go back and add.